docker/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM php:5.6-apache
+
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash
+
+RUN echo deb http://http.debian.net/debian jessie-backports main >> /etc/apt/sources.list
+
+RUN apt-get update --fix-missing && \
+    apt-get install -y \
+    zlib1g-dev \
+    libpng-dev \
+    libcurl4-openssl-dev \
+    libicu-dev \
+    libmcrypt-dev \
+    libxml2-dev \
+    libmagickwand-dev \
+    git \
+    nodejs \
+    unzip \
+    bzip2 \
+    rubygems \
+    ruby-dev \
+    php5-cli \
+    php5-common \
+    php5-sqlite \
+    rsyslog \
+    libnotify-bin \
+    sudo
+
+RUN docker-php-ext-install mbstring pdo_mysql zip gd curl intl mcrypt soap opcache json
```

Comment:
I included the following line before:

```
RUN docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/
```

Otherwise JPEG images were not processed.